api/api_test.go

```diff
@@ -107,6 +107,19 @@ func TestCompanyHandler(t *testing.T) {
 			http.StatusOK,
 			expected,
 		},
+		//Solicitação de campos específicos
+		{
+			http.MethodGet,
+			"/19.131.243/0001-97?fields=uf,cep",
+			http.StatusOK,
+			`{"cep":"","uf":""}`,
+		},
+		{
+			http.MethodGet,
+			"/19131243000197?fields=xolofompila",
+			http.StatusNotFound,
```

Comment:
Não acho que `404 Not found` descreva a situação aqui, pois o recurso existe. O problema é a requisição solicitando coisa que não existe. Acho que é o caso de um `400 Bad request`. O que você acha?

Reply:
Pode ser, na minha cabeça eu estava pensando "o campo não foi encontrado".